backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/parsers/CLIParser.java

```diff
@@ -57,8 +57,7 @@
 
     private static final List<AttachmentType> ACCEPTABLE_ATTACHMENT_TYPES = ImmutableList.of(
```

Comment:
Yes, for all practical purposes, it is superflous. What do you think would be a good place to centrally store the list of license info attachment types?